enterprise_access/apps/api/serializers/content_assignments/assignment.py

```diff
@@ -51,6 +51,16 @@ class LearnerContentAssignmentRecentActionSerializer(serializers.Serializer):
     )
 
 
+class ContentMetadataSerializer(serializers.Serializer):
+    """
+    Serializer to help return additional content metadata for assignments.  These fields should
+    map more or less 1-1 to the fields in content metadata dicts returned from the
+    enterprise-catalog `get_content_metadata` response payload.
+    """
+    start = serializers.DateTimeField()
+    normalized_metadata = serializers.DictField()
```

Comment:
[clarification/curious] Is there a need for `start` if `normalized_metadata` is returned, since `start_date` is also included in there?

Reply:
ah, sorry, to clarify: I just threw in these two without thinking much about it to test that everything is plumbed correctly.  I'll add the ones we actually need in a future revision of this PR.